pages/spots/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import SpotCard from "../../components/SpotCard";
+import Layout from "../../components/Layout";
+
+export default function Spots({ spots, toggleBookmark, addSpots }) {
+  return (
+    <Layout pageTitle="Spots">
```

Comment:
This is cool! You almost had it 😅 You need to pass along the `pageTitle` property in your `Layout` component.

```js
// Layout
export default function Layout({ pageTitle, children }) {
  return (
    <>
      <Header pageTitle={pageTitle} />
      <main>{children}</main>
      <Footer />
    </>
  );
}

// Header
export default function Header({ pageTitle }) {
  return (
    <StyledHeader>
      <Headline>{pageTitle}</Headline>
    </StyledHeader>
  );
}
```

Then you won´t need this part
```js
const { pathname } = useRouter();

  function setTitle() {
    switch (pathname) {
      case "/spots":
        return "Spots";
      case "/create":
        return "add your Spot";
      case "/favorites":
        return "Favorites";
    }
  }
  const title = setTitle();
```